connexta/admin-console
# E2E Testing

#### What does this PR do?

Adds some e2e tests and addresses some technical debt.

#### Who is reviewing it?

@tbatie 
@garrettfreibott 

#### How should this be tested?

- Start DDF
- Ensure embedded LDAP is installed
- In the ui module, run `yarn run start:test`
- Ensure all tests pass

#### Checklist:
